lib/jekyll/configuration.rb

```diff
@@ -246,20 +246,6 @@ def backwards_compatibilize
       config
     end
 
-    def fix_common_issues
```

Comment:
Please leave this method as a no-op returning `config` to avoid breaking plugins.

```ruby
# DEPRECATED.
def fix_common_issues
  self
end
```